parameter_files/fates_params_default.cdl

```diff
@@ -951,6 +952,14 @@ data:
  fates_leaf_slatop = 0.012, 0.01, 0.024, 0.012, 0.03, 0.03, 0.012, 0.03, 
     0.03, 0.03, 0.03, 0.03 ;
 
+ fates_leaf_stomatal_intercept = 1000, 1000, 1000, 1000, 1000, 1000, 1000, 
+    1000, 1000, 1000, 1000, 1000 ;
+
+ fates_leaf_stomatal_slope_ballberry = 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8 ;
+
+ fates_leaf_stomatal_slope_medlyn = 4.1, 2.3, 2.3, 4.1, 4.4, 4.4, 4.7, 4.7, 
+    4.7, 2.2, 5.3, 1.6 ;
+
```

Comment:
@Qianyuxuan looks good nice work on the updates here.   Question: I know you wrote up documentation to that is a companion to your code changes, we should make sure that makes it into the documentation for FATES/tech doc and that also includes these defaults and where they com from for Medlyn (USO).  have you shared that documentation you created on this PR yet?

Reply:
@serbinsh I haven't yet. I will drop a doc into the comment box here soon.